index.html

```diff
@@ -0,0 +1,172 @@
+            is
+            too
+            slow.
+            I explained 5 key areas which are important to grow in order for Africans to strongly stand in global market
+            competition
+            which are: Education, health, agriculture, construction, and computer programming.
+            If we manage to stay strong and competitive in those areas we will not only compete but also read the global
+            technology
+            market.
+          </p>
+        </div>
+        <div>
+          <p>Young Africans can use technology to compete on the global market? Yes.
+            But tech in Africa is steel at law level, many people need to copy with tech and the process of doing that
+            is
+            too
+            slow.
+            I explained 5 key areas which are important to grow in order for Africans to strongly stand in global market
+            competition
+            which are: Education, health, agriculture, construction, and computer programming.
+            If we manage to stay strong and competitive in those areas we will not only compete but also read the global
+            technology
+            market.
+          </p>
+        </div>
+        <div>
+          <p>Young Africans can use technology to compete on the global market? Yes.
+            But tech in Africa is steel at law level, many people need to copy with tech and the process of doing that
+            is
+            too
+            slow.
+            I explained 5 key areas which are important to grow in order for Africans to strongly stand in global market
+            competition
+            which are: Education, health, agriculture, construction, and computer programming.
+            If we manage to stay strong and competitive in those areas we will not only compete but also read the global
+            technology
+            market.
+          </p>
+        </div>
+        <div>
+          <p>Young Africans can use technology to compete on the global market? Yes.
+            But tech in Africa is steel at law level, many people need to copy with tech and the process of doing that
+            is
+            too
+            slow.
+            I explained 5 key areas which are important to grow in order for Africans to strongly stand in global market
+            competition
+            which are: Education, health, agriculture, construction, and computer programming.
+            If we manage to stay strong and competitive in those areas we will not only compete but also read the global
+            technology
+            market.
+          </p>
+        </div>
+      </div>
+    </section>
+  </div>
+  <!--
+
+    Hello Cam!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
Thanks, @mukhammadrizooff, however, the repo belongs to me therefore, I will not be able to select the Request Review and the Approved Buttons because they automatically become disabled.